controller_manager/controller_manager/spawner.py

```diff
@@ -301,6 +310,50 @@ def main(args=None):
                         )
                         return 1
 
+                if not fallback_controllers and param_file:
```

Comment:
might be helpful to document that the cmd line argument will override the param_file value (assuming that's the behaviour you want!) (forgive the driveby review during WG meeting)

Reply:
Got it! :+1: